.env

```diff
@@ -10,5 +10,6 @@ UI_PORT=3000
 OPENSEARCH_PORT=9200
 RAY_CONSOLE_PORT=8265
 JUPYTER_PORT=8888
+SSL=0
```

Comment:
What is the top-level .env file and what does it do?  Seems like a dangerous default setting for SSL.

Reply:
It sets the default environment when you run a docker compose.  If we don't do this, then people will get https if they don't do it manually, and Jon said we had to http.  I'm assuming other stuff won't read this.